odoo/addons/base/tests/test_orm.py

```diff
@@ -97,6 +98,15 @@ def test_access_filtered_records(self):
         with self.assertRaises(AccessError):
             (p1 + p2).with_user(user).unlink()
 
+    def test_call_kw(self):
```

Comment:
Can you move the method in `test_api.py`, in a class named `TestExternalAPI`?
I think it would make more sense, since it is a different API level than what is tested in this file.